src/css/content.css

```diff
@@ -0,0 +1,55 @@
+.background {
+    width: 100%;
+    position: absolute;
+    z-index: -1;
+}
+
+.main-header {
+    font-size: 1rem;
```

Comment:
We should look into creating reusable font classes so that they are consistent across the page.
Or use some kind of pre-made version á la bootstrap but much more scaled down